tests/test_ticks.c

```diff
@@ -4,23 +4,18 @@
 // Writes the 32bit value to COP0 register 11 (compare)
 #define WRITE_COMPARE(val) do { asm volatile("mtc0 %0,$11" ::"r"(val)); } while(0)
 
-#define ARM_INTERRUPT(compare) do { \
-	asm volatile(".align 4"); \
-	WRITE_TICKS(0x0); \
-	WRITE_COMPARE(compare); \
-} while(0)
-
 #define TEST(name, tests) do { \
 	const int NUM_CASES = sizeof(tests) / sizeof(tests[0]); \
 	for (int i=0; i < NUM_CASES; i++) { \
-		mock_tick = tests[i][1]; \
+		/* move ticks just before 5ms of the target to make sure wait is actually waiting */ \
+		mock_tick = tests[i][1] - 0x39387; \
```

Comment:
I wonder if we should have a `TICKS_FROM_MS()` macro as well. I know we have `wait_ms()` but this code proves us that we're now giving out all the helpers that we require to write the code without magic numbers.

This line would become `mock_tick = tests[i][1] - TICKS_FROM_MS(5)`